src/GenerateProgrammaticPipelineFromConfig/Generator.php

```diff
@@ -241,32 +241,20 @@ private function generateRoutes(array $config)
                 $route = empty($spec['name'])
                     ? sprintf(self::TEMPLATE_ROUTED_NO_METHOD_NO_NAME, $path, $middleware)
                     : sprintf(self::TEMPLATE_ROUTED_NO_METHOD_WITH_NAME, $path, $middleware, $spec['name']);
-
-                goto options;
```

Comment:
I know many folks do not like goto, but in this case, it eliminated a lot of code duplication. I find the revised version quite more cumbersome to read, due to additional nesting and duplication of the `$route` assigment ternary across three conditions.